tests/src/buildApplication/generateFunctionsMap.test.ts

```diff
@@ -36,14 +36,10 @@ describe('generateFunctionsMap', async () => {
 			false
 		);
 		expect(invalidFunctions.size).toEqual(0);
-		const expectedFunctions = ['index', 'index.rsc', 'api/hello'];
-		expect(functionsMap.size).toEqual(expectedFunctions.length);
-		expectedFunctions.forEach(fn => {
-			expect(functionsMap.has(fn)).toBe(true);
-			const path = functionsMap.get(fn);
-			const expectedFnPathRegex = new RegExp(`/${fn}.func.js$`);
-			expect(path).toMatch(expectedFnPathRegex);
-		});
+		expect(functionsMap.size).toEqual(3);
+		expect(functionsMap.get('index')).toMatch(/\/index.func.js$/);
+		expect(functionsMap.get('index.rsc')).toMatch(/\/index.rsc.func.js$/);
+		expect(functionsMap.get('api/hello')).toMatch(/\/api\/hello.func.js$/);
```

Comment:
I'm being pedantic now 😄 

```suggestion
		expect(functionsMap.get('index')).toMatch(/\/index\.func\.js$/);
		expect(functionsMap.get('index.rsc')).toMatch(/\/index\.rsc\.func\.js$/);
		expect(functionsMap.get('api/hello')).toMatch(/\/api\/hello\.func\.js$/);
```

Right?

Reply:
yeah not pedantic at all, sorry I missed those 😓 
Thanks a lot for noticing them! 🙏